src/main/java/org/example/models/Capability.java

```diff
@@ -0,0 +1,35 @@
+package org.example.models;
+
+public enum Capability {
```

Comment:
making this an enum means we'll have to make a code change in the future if someone adds a new capability which we don't want to do. just use a string for capability